controllers/openstack_ansibleee_controller.go

```diff
@@ -99,11 +99,18 @@ func (r *OpenStackAnsibleEEReconciler) Reconcile(ctx context.Context, req ctrl.R
 			}
 			if c.Status != corev1.ConditionTrue {
 				ready = false
+				if c.Reason == condition.ErrorReason {
+					instance.Status.Conditions.MarkFalse(
+						condition.ReadyCondition,
+						condition.ErrorReason,
+						condition.SeverityError,
+						c.Message)
+				}
```

Comment:
So this basically copies the message of the first non True condition from the conditions list into the Message of the ReadyCondition. We have a more detailed approach in lib-common's MirrorCondition call. That will use the most sever and latest non True condition.

Reply:
in case of error, don't we want the first one? As it is probably the one that is causing the others to fail

Reply:
I tried using mirror, but the status didn't get updated
```
oc get openstackansibleee -w -o wide                             
NAME                       NETWORKATTACHMENTS   STATUS   MESSAGE
ansibleee-playbook-local                                 
ansibleee-playbook-local                        Unknown   Setup started
ansibleee-playbook-local                        Unknown   Setup started
ansibleee-playbook-local                        Unknown   Setup started
```

with the current code:
```
oc get openstackansibleee -w -o wide
NAME                       NETWORKATTACHMENTS   STATUS   MESSAGE
ansibleee-playbook-local                                 
ansibleee-playbook-local                        Unknown   Setup started
ansibleee-playbook-local                        False     AnsibleExecutionJob error occured job.name ansibleee-playbook-local job.namespace default failed
ansibleee-playbook-local                        False     AnsibleExecutionJob error occured job.name ansibleee-playbook-local job.namespace default failed
```

Reply:
actually, I was doing it wrong, fixed it now